Add mysql/ to PATH_SUFFIXES for the library search for mysql.

On common distros libmysqlclient* is installed and/or symlinked
into /usr/lib.
Since we install the libraries into /usr/lib/mysql alone cppdb
fails to properly link to it giving runtime errors.

Status: https://sourceforge.net/p/cppcms/bugs/135/

Signed-off-by: Gustavo Zacarias <gustavo@zacarias.com.ar>

diff -Nura cppdb-0.3.1.orig/CMakeLists.txt cppdb-0.3.1/CMakeLists.txt
--- cppdb-0.3.1.orig/CMakeLists.txt	2014-10-17 11:05:57.875429748 -0300
+++ cppdb-0.3.1/CMakeLists.txt	2014-10-17 11:12:57.486775484 -0300
@@ -196,7 +196,7 @@
 endif()
 
 if(NOT DISABLE_MYSQL)
-	find_library(MYSQL_LIB mysqlclient)
+	find_library(MYSQL_LIB mysqlclient PATH_SUFFIXES mysql)
 	find_path(MYSQL_PATH mysql.h PATH_SUFFIXES mysql)
 	if(MYSQL_LIB AND MYSQL_PATH)
 		include_directories(${MYSQL_PATH})
